importer/src/main/kotlin/com/ritense/importer/ValtimoImportService.kt

```diff
@@ -142,42 +144,62 @@ open class ValtimoImportService(
 
     }
 
-//    @Transactional
-//    open fun importResources(resources: Array<Resource>) {
-//        val entries = getEntriesFromResources(resources)
-//        val importerEntriesMapGroupedByCaseDefinition = entries.map {
-//            it.key to getEntriesByImporter(it.value)
-//        }.toMap()
-//
-//        importerEntriesMapGroupedByCaseDefinition.forEach { (basePath, importerEntriesMap) ->
-//            var caseDefinitionId: CaseDefinitionId? = null
-//            if (basePath?.isNotEmpty() == true) {
-//                val caseDefinitionEntries = importerEntriesMap
-//                    .filter { it.key.type() == CASE_DEFINITION }
-//                    .let {
-//                        it[it.keys.first()]
-//                    }
-//
-//                val caseDefinitionMap: Map<String, Any> = jacksonObjectMapper()
-//                    .readValue(caseDefinitionEntries?.first()?.content!!)
-//                caseDefinitionId = CaseDefinitionId(
-//                    caseDefinitionMap["key"] as String,
-//                    caseDefinitionMap["versionTag"] as String
-//                )
-//            }
-//
-//            importerEntriesMap.forEach { (importer, entries) ->
-//                entries.forEach { entry ->
-//                    logger.debug { "Importing ${entry.fileName} with importer ${importer.type()}" }
-//                    importer.import(ImportRequest(entry.fileName, entry.content, caseDefinitionId))
-//                }
-//            }
-//        }
-//    }
-
     @Transactional
-    override fun import(inputStream: InputStream) {
+    override fun import(inputStream: InputStream, caseDefinitionIdList: List<CaseDefinitionId>) {
```

Comment:
Isn't this very similar to the other method in this class. can't we combine them somehow?